source/elements/oneMKL/source/domains/rng/device_routines/device-distributions-template-parameter-method.rst

```diff
@@ -0,0 +1,49 @@
+.. SPDX-FileCopyrightText: 2023 Intel Corporation
+..
+.. SPDX-License-Identifier: CC-BY-4.0
+
+.. _onemkl_device_rng_uniform_continuous_distributions_method:
+
+Distributions Template Parameter Method
+=======================================
+
+.. tabularcolumns:: |\Y{0.4}|\Y{0.2}|\Y{0.4}|
+
+.. list-table::
+   :header-rows: 1
+   :class: longtable
+
+   * -  Method Type
+     -  Distributions
+     -  Math Description
+
+   * -  ``uniform_method::standard``
+        ``uniform_method::accurate``
+     -  ``uniform``
+     -   Standard method. Currently there is only one method for these functions. ``uniform_method::accurate`` checks for additional ``s`` and ``d`` data types. For ``integer`` data types, it uses ``d`` as a ``BRNG`` data type (``s`` ``BRNG`` data type is used in ``uniform_method::standard`` method on GPU).
+   * -  ``gaussian_method::box_muller2``
+     -  ``gaussian``
+     -  Generates normally distributed random numbers x1 and x2 thru the pair of uniformly distributed numbers u1 and u2 according to   the formulas: \ :math:`x_1 = \sqrt{-2 \ln u_1} \sin {2 \pi u_2}`\       \ :math:`x_2 = \sqrt{-2 \ln u_1} \cos {2 \pi u_2}`\       Lognormal   distribution: generated normally distributed random numbers x1 and x2   are converted to lognormal distribution.
+   * -  ``exponential_method::icdf``
+        ``exponential_method::icdf_accurate``
+     -  ``exponential``
+     -  Inverse cumulative distribution function (ICDF) method.
```

Comment:
No description of ICDF accurate

Reply:
I'm afraid we don't have description for icdf method anywhere else either. I suggest to leave it as it is for now and add later